Demo/Demo/UseCases/GetOrderIdUseCase.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  GetOrderIdUseCase.swift
+//  Demo
+//
+//  Created by Jose Noriega on 31/08/2022.
+//
+
+import UIKit
+
+final class GetOrderIdUseCase {
```

Comment:
@jaxdesmarais is it time to rethink this convention we're following? in nxo lib all lables are like `orderId` instead of `orderID` so we might end up with these two in many places

Reply:
NXO is an external library so can follow their own style guide which may be different by our own. That said, they do use both the Id (legacy NXO code) and ID (newer code). Talking to some folks on NXO, moving forward their style guide also indicates using `ID` for all new code but this would be a breaking change for their merchants so they haven't yet made a change for legacy code as that would need to be done in a major version.